_elements/buttons.md

```diff
@@ -70,7 +70,11 @@ lead: Use buttons to signal actions.
   </button>
   <div id="collapsible-0" aria-hidden="false" class="usa-accordion-content">
     <h4 class="usa-heading">Accessibility</h4>
-    <p>No specific accessibility guidelines.</p>
+    <ul>
+      <li>Buttons should display a "halo" when users tab to them.</li>
```

Comment:
How about "Buttons should display a visible focus state when users tab to them." ?